dataprep/eda/correlation/compute/__init__.py

```diff
@@ -3,19 +3,21 @@
 
 from typing import Optional, Tuple
 
+from ...configs import Config
 from ...data_array import DataArray, DataFrame
+from ...dtypes import NUMERICAL_DTYPES
 from ...intermediate import Intermediate
+from ...utils import to_dask
 from .bivariate import _calc_bivariate
 from .nullivariate import _calc_nullivariate
 from .univariate import _calc_univariate
-from ...dtypes import NUMERICAL_DTYPES
-from ...utils import to_dask
 
 __all__ = ["compute_correlation"]
 
 
 def compute_correlation(
```

Comment:
Can we keep the same parameter order? I see some functions have cfg right after df, some have it after x and y. 
How about
```python
func(df: DataFrame, x:  ..., y: ..., *, cfg: Config, dtype:...)
```
?

Reply:
The problem is a non-default argument (`config`) cannot follow a default argument (`x`). I'm thinking it might be better to have 
``` python
def compute_missing(
    df: DataFrame,
    x: Optional[str] = None,
    y: Optional[str] = None,
    *
    cfg: Optional[Config] = None,
    display: Optional[List[str]] = None,
    dtype: Optional[DTypeDef] = None,
)
```
So that the user can call `compute_*()` without instantiating a config class. What do you think?

Reply:
I think since we already have default values in the Config class, we can instantiate a default config if cfg is None.